### ISSUE TITLE
Cannot manually change the scene numbering for special episodes

### ISSUE THREAD

**Issue by duramato** [pymedusa/Medusa#1451]
**Branch/Commit:** develop
**OS:** Ubuntu 14.04
**What you did:** Tried changing the scene numbering of the special of [this show](http://thetvdb.com/?tab=series&id=302621) to 2x0 it always reverts back to the indexer numbering.
**What happened:** It rechanges back to the indexer numbering
**What you expected:** For it to change to the numbering i input like it should

@p0psicles this seems to be the same js issue you had fixed alooong time ago in upstream SR

### Visual feedback of the normal (working ep) and the special (not working), respectivly: 

Both show the "green feeback" as in if it where sucessfull, when in fact only the first was.
![image](https://cloud.githubusercontent.com/assets/10121067/19948896/dac14a8a-a146-11e6-83e4-0f620c963dc6.png)
![image](https://cloud.githubusercontent.com/assets/10121067/19948899/dd976442-a146-11e6-868b-d1d39dfc109e.png)

### Responses of "/setSceneNumbering?show=302621&indexer=1&forSeason=1&forEpisode=4&sceneSeason=2&sceneEpisode=0" for each episode, respectivly
```
{"forEpisode": "1", "sceneEpisode": 1, "sceneSeason": 0, "success": true, "forSeason": "0"}
```

```
{"forEpisode": "4", "sceneEpisode": 0, "sceneSeason": 2, "success": true, "forSeason": "1"}
```

**Comment by p0psicles** [comment]
@omg can you pick up this one?

**Comment by p0psicles** [comment]
Aah it's the famous 'if season and episodes (should be is not None) bug'

**Comment by p0psicles** [comment]
Fixed by #1485
